src/components/application_manager/test/message_helper/message_helper_test.cc

```diff
@@ -957,6 +976,422 @@ TEST_F(MessageHelperTest, SubscribeApplicationToSoftButton_CallFromApp) {
       message_params, appSharedPtr, function_id);
 }
 
+TEST_F(MessageHelperTest, SendOnAppUnregNotificationToHMI) {
+  const bool kIsUnexpectedDisconnect = false;
+
+  MockApplicationSharedPtr mock_app = ::utils::MakeShared<AppMock>();
+
+  uint32_t mock_hmi_app_id = 1u;
```

Comment:
@AByzhynar Maybe I am wrong, but on the next line there is `EXPECT_CALL` which performs `RETURN()` on this variable, so I don't consider it to be const.